<?xml version="1.0"?>
<launch>
  <node name="gpio" pkg="teapicking" type="gpio.py" output="screen">  
  </node>
  <arg name="joy_dev" default="/dev/input/js0" />
  <node pkg="joy" type="joy_node" name="joy_node">
    <param name="dev" value="$(arg joy_dev)" />
    <param name="deadzone" value="0.05" />
    <param name="autorepeat_rate" value="0" />
  </node>
  <node name="joy_lai" pkg="teapicking" type="joy_lai.py" output="screen">  
  </node>
  <node name="http_lai" pkg="teapicking" type="http_lai.py" output="screen">  
  </node>
  <node name="launch" pkg="teapicking" type="launch.py" output="screen">  
  </node>
  <!--node name="launch_stop" pkg="teapicking" type="launch_stop.py" output="screen">  
  </node-->
 
</launch>
